diffpy/nmf_mapping/nmf_mapping/main.py

```diff
@@ -37,7 +37,7 @@ def tup(s):
         try:
             l, h = map(int, s.split(","))
             return l, h
-        except:
+        except Exception:
```

Comment:
this is looking a bit ugly.  Why are we changing `Exception` into `TypeError`?  I expect there is a better way of doing this.